### DISCUSSION TITLE
The generated hive DDL of Adding column is wrong

### DISCUSSION THREAD

**Discussion by AlexLuya** [dbeaver/dbeaver#257] [Q&A] [answered]
If trying to add a column,right ddl must be 

``` sql
ALTER TABLE table ADD COLUMNS (newColumn String COMMENT 'comments')
```

but dbeaver will generate 

``` sql
ALTER TABLE test.table ADD newColumn VARCHAR(100);
```

**Comment by serge-rider** [comment]
What database? Different databases have different syntax.

**Comment by AlexLuya** [comment]
hive

**Comment by serge-rider** [comment, marked as answer]
Unfortunately DBeaver doesn't support Hive-specific DDL (yet, maybe).
Generated DDL conforms SQL-92 standard.
To generate Hive-specific DDL we'll need to make new driver extension, at the moment there are no plans to make it for Hive (or there are too many other more prioritized databases to support).
You can make feature request for Hive extension. But I can't say when I'll have a time to work on it...

**Comment by AlexLuya** [comment]
I think if you don't remove hive support,this bug should be kept open

**Comment by serge-rider** [comment]
If I will keep all not-implemented features for all unsupported databases open then there will be zillions of open issues with blurry status. It is hard to manage big number of open issues.
I think it's better to reopen them or create new ones when it will make sense.
